### ISSUE TITLE
Covert to Redux Sagas

### ISSUE THREAD

**Issue by jblevins1991** [nickstaroba/react-redux-example#4]
Hey, just got done looking through your codebase. Thought I would make a recommendation based on what I have been using on projects in my free time. Redux sagas solves a problem that is complex and timing consuming to solve with Thunk. It takes the state from only the latest copy of a specific action. This solves the issue of cancelling https requests (as the abort signal on fetch and other xhr libraries add a lot of complexity to a project).

**Comment by nickstaroba** [comment]
Yeah I've been meaning to check that out. This codebase is just an example I used to understand the redux pattern of a project I'm working on. I made it public to share with a friend so they could understand the pattern as well.
